internal/pkg/egressacl/yaml_test.go

```diff
@@ -0,0 +1,80 @@
+package egressacl
+
+import "github.com/stretchr/testify/assert"
+import "github.com/stripe/smokescreen/pkg/egressacl/decision"
+import "testing"
```

Comment:
I don't think this is gofmt'd correctly?

Reply:
It did!